### PR TITLE
Css canvas medical -Lisa Le

### PR DESCRIPTION
**Pull request by lisale841**
https://user-images.githubusercontent.com/93673754/145174665-999fff23-1ec6-4f82-8989-9047630282a3.mov

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <div class="full-column">
+            <div class="bottom-links">
+              <div class="bottom-links-container">
+                <a href="">Privacy & Terms</a>
+                <a href="">Contact Us</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: It looks like you accidentally made each of these `row`s a child of the `row` element for the first `row` 🤔 . You should revise this and see if it has any effect on your output

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Lisa, great start on this! One note to get you started:

File: css-canvas-medical/index.html (hunk 1 above)
Comment (lisale841) [reply css-canvas-medical/index.html]: hi cody.  Thank you.  I just revised it.

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**